orcid-core/src/main/java/org/orcid/core/manager/impl/EmailMessageSenderImpl.java

```diff
@@ -343,13 +343,7 @@ public void sendEmailMessages() {
                 
                 if(!notifications.isEmpty()) {
                     LOGGER.info("Found {} messages to send for orcid: {}", notifications.size(), orcid);
-                    EmailMessage digestMessage;
-                    if(Features.VERBOSE_NOTIFICATIONS.isActive()) {
-                        digestMessage = createDigest(orcid, notifications);
-                    } else {
-                        digestMessage = createDigestLegacy(orcid, notifications);
```

Comment:
@auumgn looks like you can also delete the function `createDigestLegacy` since no one use it anymore.